graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -108,107 +135,139 @@ public void upgrade() {
                 "/usr/share/filebeat/bin/filebeat",
                 "-c  %s",
                 "test config -c %s",
-                beatsPreambel +
-                        "filebeat.inputs:\n" +
-                        "- input_type: log\n" +
-                        "  paths:\n" +
-                        "    - /var/log/*.log\n" +
-                        "  type: log\n" +
-                        "output.logstash:\n" +
-                        "   hosts: [\"192.168.1.1:5044\"]\n" +
-                        "path:\n" +
-                        "  data: ${sidecar.spoolDir!\"/var/lib/graylog-sidecar/collectors/filebeat\"}/data\n" +
-                        "  logs: ${sidecar.spoolDir!\"/var/lib/graylog-sidecar/collectors/filebeat\"}/log"
+                f("""
+                                %s
+                                filebeat.inputs:
+                                - input_type: log
+                                  paths:
+                                    - /var/log/*.log
+                                  type: log
+                                output.logstash:
+                                   hosts: ["${user.graylog_host}:5044"]
+                                path:
+                                  data: ${sidecar.spoolDir!"/var/lib/graylog-sidecar/collectors/filebeat"}/data
+                                  logs: ${sidecar.spoolDir!"/var/lib/graylog-sidecar/collectors/filebeat"}/log""",
+                        beatsPreambel
+                )
         );
+        ensureCollector(
+                "auditbeat",
+                "exec",
+                "linux",
+                "/usr/lib/graylog-sidecar/auditbeat",
+                "-c  %s",
+                "test config -c %s",
+                f("""
+                                %s
+                                auditbeat.modules:
+                                - module: file_integrity
+                                  paths:
+                                  - /bin
+                                  - /usr/bin
+                                  - /sbin
+                                  - /usr/sbin
+                                  - /etc
+                                output.logstash:
+                                   hosts: ["${user.graylog_host}:5044"]
+                                path:
+                                  data: /var/lib/graylog-sidecar/collectors/auditbeat/data
+                                  logs: /var/lib/graylog-sidecar/collectors/auditbeat/log""",
+                        beatsPreambel)
+        ).ifPresent(collector -> ensureDefaultConfiguration("default-linux", collector));
```

Comment:
```suggestion
        ).ifPresent(collector -> ensureDefaultConfiguration("auditlogbeat-default", collector));
```
I'd rather name that after the collector. the OS is implicit. actually the collector is too, but I think it looks better this way